extensions/sql-assessment/src/tabs/assessmentMainTab.ts

```diff
@@ -106,12 +106,18 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 				dark: this.extensionContext.asAbsolutePath('resources/dark/database.svg'),
 				light: this.extensionContext.asAbsolutePath('resources/light/database.svg')
 			};
+		const iconSize: number = 16;
 
 		const btnInvokeAssessment = view.modelBuilder.button()
 			.withProperties<azdata.ButtonProperties>({
-				label: this.invokeAssessmentLabel,
+				label: this.invokeAssessmentLabel
+			})
+			.withProperties<azdata.ComponentWithIconProperties>({
```

Comment:
Just add these to the first withProperties function, no reason to split them up like this (same goes for all these)